CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Fixed an issue in the **download** command where layouts were overriden even without the `-f` option.
```

Comment:
add another bullet for the actual bug, where the SDK didn't appropriately detect the `id` (so that if anything happens down the road, we'll notice it in the changelog)